### PR TITLE
Removes batch size option on request

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: lib/export.js [hunk 1]
```diff
@@ -150,14 +150,16 @@ module.exports = (self) => {
 
             self.cleanup(filepath);
 
+            const expiration = self.options.export && self.options.export.expiration;
```
Comment (boutell) [inline lib/export.js:153]: This needs a reasonable default.

File: index.js [hunk 2]
```diff
@@ -71,8 +71,10 @@ module.exports = {
           req.body._ids = self.apos.launder.ids(req.body._ids);
 
           const extension = self.apos.launder.string(req.body.extension);
-          const batchSize = self.apos.launder.integer(req.body.batchSize);
-          const expiration = typeof self.options.export === 'object' ? self.apos.launder.integer(self.options.export.expiration) : null;
+          const batchSize = typeof self.options.export === 'object' &&
```
Comment (boutell) [inline index.js:74]: laundering module config options is not necessary. Both of these need reasonable defaults.

**Review by boutell — CHANGES_REQUESTED** [review]

File: lib/export.js (hunk 1 above)
Comment (abea) [reply lib/export.js:153]: Instead of using the default on L162?

**Review by boutell — APPROVED** [review]
My bad.

**Pull request merged**